src/backend/TrafficCourts/Citizen.Service/Models/Dispute/ViolationTicket.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class ViolationTicket
```

Comment:
We already have [TrafficCourts.Citizen.Service.Models.Tickets.ViolationTicket](https://github.com/bcgov/jag-traffic-courts-online/blob/main/src/backend/TrafficCourts/Citizen.Service/Models/Tickets/ViolationTicket.cs) that has comments